Gemfile

```diff
@@ -3,14 +3,14 @@ source 'http://rubygems.org'
 ruby '2.2.3'
 
 group :development do
-  gem 'activesupport'
-  gem 'cucumber'
-  gem 'cucumber_lint'
-  gem 'kappamaki'
-  gem 'mortadella'
-  gem 'open4'
-  gem 'parallel_tests'
-  gem 'rake'
-  gem 'rubocop'
-  gem 'rspec'
+  gem 'activesupport', '4.2.6'
```

Comment:
In my opinion this is an anti-pattern in Ruby. The versions are locked down in Gemfile.lock. The Gemfile should only lock down versions if there is a good reason for staying on that version, even if other dependencies get updated. Like "version X of this gem is broken, lets avoid it".